habitat-lab/habitat/config/habitat/task/rearrange/ovmm.yaml

```diff
@@ -87,15 +100,27 @@ measurements:
     ee_resting_success_threshold: 100
   ovmm_place_success:
     ee_resting_success_threshold: 100
-    check_stability: False
+    check_stability: True
   dist_to_pick_goal:
     use_shortest_path_cache: False # when caching, the same set of goals are used throughout
   dist_to_place_goal:
     use_shortest_path_cache: False # when caching, the same set of goals are used throughout
   nav_orient_to_pick_succ:
     must_call_stop: False
+    success_angle_dist: 3.14
+    min_object_coverage_iou: 1e-3
   nav_orient_to_place_succ:
     must_call_stop: False
+    success_angle_dist: 3.14
```

Comment:
I am not using angle-based success for nav to object success. I am checking for coverage of object in agent's camera view. This is set to high so that angle-based success doesn't result in failures.

Reply:
Was this added by us or Andrew?

Reply:
`success_angle_dist` was there from before. This is the maximum error in the robot's orientation to the object's center. I will add a config option to replace this with the IoU coverage-based metric. Currently, both conditions are present, although with 3.14 angle-based one always passes.